modules/cli/src/main/scala/scala/cli/commands/publish/Ivy.scala

```diff
@@ -38,9 +37,9 @@ object Ivy {
       Option[Configuration],
       MinimizedExclusions
     )] = Nil,
-    license: Option[License] = None,
-    scm: Option[Scm] = None,
-    developers: Seq[Developer] = Nil,
+    // TODO: license: Option[License] = None,
```

Comment:
Do we have an issue to track it? I would mention it here

Reply:
- https://github.com/VirtusLab/scala-cli/issues/3914
```suggestion
    // https://github.com/VirtusLab/scala-cli/issues/3914
    // TODO: license: Option[License] = None,
```